arches_modular_reports/settings.py

```diff
@@ -270,17 +270,6 @@
     },
 }
 
-REST_FRAMEWORK = {  # if you are using the Django REST Framework integration
```

Comment:
This is no longer needed (and no longer mentioned in the querysets install instructions)